### ISSUE TITLE
Plese don't use country flags to refere to languages

### ISSUE THREAD

**Issue by arobase-che** [valeriansaliou/sonic#297]
Hi, \o
There is a long list of arguments about this. As a Frenchman, I refuse to let my flag be used to exclude most French speakers.

The same goes for the Portuguese. Portugal is a country where Portuguese is the official language, but it is only the 4th most populous country (Brazil, Angola and Mozambique are more populous), so it is not appropriate to use Portuguese to refer to the language. Idem 

Please, just don't.

Related: #210

**Comment by MichaelBagnasco** [comment]
I disagree. Flags are instantly recognizable symbols that cross language barriers. 

Say you're Chinese, and you don't read English well. You see a list of flags, and spot the Chinese flag, and you instantly know that your language is supported. 

Also, using flags to represent language is a nearly ubiquitous user interface design choice.

**Comment by valeriansaliou** [comment]
Definitely, it's definitely became a standard nowadays to refer to languages w/ their associated country flag. We can't satisfy everyone with that, but at least UIs get more clarity that way. I won't change anything on that in the README.

**Comment by arobase-che** [comment]
Thank you for your reply, but I still don't agree.
It is not a standard to use flags instead of language. Please provide a reference website that actually uses flags to represent the language (not the country!).

Regarding not using flags:
 - Netflix doesn't
 - Facebook doesn't
 - Amazon doesn't use it (but that's to refer to the country)
 - Mozilla doesn't use it
 - Youtube doesn't use it
 - Apple doesn't
 - Ubuntu doesn't
 
 It's actually the contrary, **not using flags is the standard**. And by standard, I mean of the web. It's in the [FAQ of the w3c about languages](https://www.w3.org/International/questions/qa-link-lang#flags).
 
 I insist on the exclusionary nature of using flags to represent language but in the end obviously, it's your repository.
 
 @MichaelBagnasco I'm sorry but your example is just bad. A Chinese person will have to wonder if it's simplified or traditional Chinese (or even mandarin sometimes). The good way to do it is actually to write the language in the language of chose. For example Facebook use "中文(简体)". You can guess that the round bracket indicate that it's simplified Chinese.

**Comment by RemiBardon** [comment]
I was thinking about this and realized we’ll have no reason to list languages in the README anymore after we remove hardcoded stopwords (#379). Only scripts will have a meaning (for tokenization, via `charabia` (see #375)); languages won’t be detected by Sonic anymore. Closing this as “Not planned”.
